datasets/_global/gleif/crawler.py

```diff
@@ -290,6 +290,12 @@ def parse_rr_file(context: Context, fh: BinaryIO):
         proxy.add("role", rel_type.replace("_", " "))
         proxy.add("status", rel.findtext("RelationshipStatus"))
 
+        if rel_schema == "Ownership":
+            # Organization entities cannot be assets, so we re-emit the asset entity as a Company
+            asset = context.make("Company")
+            asset.id = lei_id(start_lei)
```

Comment:
I wonder if it's worth adding an assertion or something that `start_prop == "asset"` - basically - if we edit `RELATIONSHIPS`, will we think to check what makes assumptions about the ordering of the props?

Reply:
I think a typesafe FtM-python would be really nice.